framework/src/mesh/FaceInfo.C

```diff
@@ -39,11 +39,11 @@ FaceInfo::FaceInfo(const Elem * elem, unsigned int side, const Elem * neighbor)
     _neighbor_centroid(_valid_neighbor ? neighbor->vertex_average()
                                        : 2 * (_face_centroid - _elem_centroid) + _elem_centroid),
     _neighbor_volume(_valid_neighbor ? neighbor->volume() : _elem_volume),
-    _gc((_neighbor_centroid - _face_centroid).norm() /
-        ((_neighbor_centroid - _face_centroid).norm() + (_elem_centroid - _face_centroid).norm())),
     _d_cf(_neighbor_centroid - _elem_centroid),
     _d_cf_mag(_d_cf.norm()),
-    _e_cf(_d_cf / _d_cf_mag)
+    _e_cf(_d_cf / _d_cf_mag),
+    _gc((_neighbor_centroid - _face_centroid).norm() /
+        ((_neighbor_centroid - _face_centroid).norm() + (_elem_centroid - _face_centroid).norm()))
```

Comment:
no need to change those

Reply:
Ah yes, I left it like that since at some point I experimented with skewness-dependent gc-s and they needed to be initialized after e_cf. I'll change it back in the next commit.